pages/myposts.vue

```diff
@@ -244,314 +255,250 @@
     </b-container>
   </client-only>
 </template>
-<script>
+
+<script setup>
 import { useRoute } from 'vue-router'
 import pluralize from 'pluralize'
 import dayjs from 'dayjs'
-import { useGroupStore } from '../stores/group'
 import { useAuthStore } from '../stores/auth'
 import { useMessageStore } from '../stores/message'
 import { useMiscStore } from '../stores/misc'
-import { useComposeStore } from '../stores/compose'
 import { useSearchStore } from '../stores/search'
-import VisibleWhen from '~/components/VisibleWhen'
 import { buildHead } from '~/composables/useBuildHead'
+
+import VisibleWhen from '~/components/VisibleWhen'
```

Comment:
Had to get rid of all the dynamically-imported components: Nuxt seems to break when used in the script setup. Can research more on the topic, or we can leave it as this.

FYI, in other components which have already been rewritten to use script setup, I don't see any dynamically-imported components.

Reply:
I think the components importing works better in this model, so the dynamic import is not needed.